docs/components/light.md

```diff
@@ -153,9 +153,11 @@ omni-directional. They mainly cast light in one direction, like the
 
 ### Probe
 
+[cubemap]: ../primitives/a-cubemap.md
+
 Probe lights are kind of like ambient lighting in that they don't have a particular source or direction and light everything equally.
 
-Where they differ though is that they will color each angle differently based upon a spherical harmonic. This spherical harmonic is generated by analyzing a cube map. The cube map you provide doesn't need to be high resolution since it's only used to generate the spherical harmonics.
+Where they differ though is that they will color each angle differently based upon a spherical harmonic. This spherical harmonic is generated by analyzing a [cubemap][cubemap]. The cubemap you provide doesn't need to be high resolution since it's only used to generate the spherical harmonics.
```

Comment:
We should be consistent in usage "cube map" vs. "cubemap" - I standardized on the version used elsewhere.